crypto/rsa/rsa_asn1.c

```diff
@@ -172,14 +173,14 @@ RSA *RSA_parse_private_key(CBS *cbs) {
     goto err;
   }
 
-  if (!parse_integer(&child, &ret->n) ||
-      !parse_integer(&child, &ret->e) ||
-      !parse_integer(&child, &ret->d) ||
-      !parse_integer(&child, &ret->p) ||
-      !parse_integer(&child, &ret->q) ||
-      !parse_integer(&child, &ret->dmp1) ||
-      !parse_integer(&child, &ret->dmq1) ||
-      !parse_integer(&child, &ret->iqmp)) {
+  if (!parse_integer_consttime(&child, &ret->n) ||
+      !parse_integer_consttime(&child, &ret->e) ||
```

Comment:
...as discussed previously (on IRC, IIRC), we don't want `n` and `e` to have the consttime flag.